app/views/posts/index.html.erb

```diff
@@ -1 +1,100 @@
-<h2>Lists of all posts</h2>
\ No newline at end of file
+<% @title='My Blog | Posts' %>
+<div class="mt-3 d-flex align-items-center flex-column">
+  <div class="card mb-3 border-primary" style="width: 680px">
+    <div class="row g-0">
+      <div class="col-md-3">
+        <img
+          src="<%= @user.photo %>"
+          class="img-fluid rounded-start"
+          alt="..."
+        />
+      </div>
+      <div class="col-md-4">
+        <div class="card-body">
+          <h2 class="card-title"><%= @user.name %></h2>
+        </div>
+        <div class="d-flex justify-content-end" style="width: 350px">
+          <p class="card-text">
+            <small class="text-muted"
+              >Number of posts: <%= @user.posts_counter %></small
```

Comment:
- [ ]  kindly change `posts_counter` to `postsCounter` as it is what you used in your schema